src/components/crud-form-pages/NoVpcTile.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { CloudAlerting } from "@carbon/icons-react";
+import { Tile } from "@carbon/react";
+import "./no-secrets-manager-tile.css";
+
+export const NoVpcTile = (resource) => {
+  return (
+    <Tile className="tileBackground displayFlex alignItemsCenter wrap marginTop">
+      <CloudAlerting size="24" className="iconMargin" /> No VPCs have
+      been created. Create one from the
```

Comment:
Can we copy the message on the Power Page? 

So something like "No VPCs have been created. Return to the Virtual Private Clouds Page."

@jvallexm we also need to add an issue to keep the headers on the Power pages even if the message is shown.
<img width="862" alt="Screenshot 2023-10-17 at 12 19 30 PM" src="https://github.com/IBM/icse-react-assets/assets/11401353/7299c1da-bf28-4454-b6ca-5b35e4c2ce7a">

Reply:
yeah we need a new issue for that

Reply:
@evpirnia Going to say ```"No VPCs have been created. Return to the Virtual Private Clouds Page to enable {resource}."``` Is that what you mean?

Reply:
sure!